src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```diff
@@ -62,6 +67,52 @@ th {
   padding-block: 0 !important;
 }
 
+//  Overriding badges to match the class name of tags
```

Comment:
let's differentiate the section like the others
```suggestion
/******************************************
Badges
******************************************/
```

Reply:
sure i will make this changes